charts/elasticsearch-index/values.yaml

```diff
@@ -8,3 +8,6 @@ hook:
 # either a 'post'-subchart to an elasticsearch chart or a 'pre'-subchart to wire-server
   enabled: true
   type: pre # possible values: 'pre' or 'post'
+image:
+  repository: quay.io/wire/brig-index
+  tag: 2.69.0
```

Comment:
This is not the latest version, but I chose this to match versions of other images on `develop`.

Reply:
This version will actually not work, will it? As in, the 2.69 will fail with the above config?

Reply:
Hmm, yes it will not work. I guess we'll have to make a commit with all versions bumped after the new docker images are generated.

Reply:
After https://github.com/wireapp/wire-server/pull/1023 got merged, this works with brig-index-2.77.3. So, this PR is now ready to be reviewed and merged.